storage/remote/write_handler_test.go

```diff
@@ -188,19 +476,98 @@ func TestOutOfOrderHistogram(t *testing.T) {
 
 	for _, tc := range tests {
 		t.Run(tc.Name, func(t *testing.T) {
-			buf, _, _, err := buildWriteRequest(nil, []prompb.TimeSeries{{
+			payload, _, _, err := buildV2WriteRequest(nil, []writev2.TimeSeries{{
+				LabelsRefs: []uint32{0, 1},
+				Exemplars:  []writev2.Exemplar{{LabelsRefs: []uint32{2, 3}, Value: 1, Timestamp: tc.Timestamp}},
+			}}, []string{"__name__", "metric1", "foo", "bar"}, nil, nil, nil, "snappy") // TODO(bwplotka): No empty string!
```

Comment:
what does your todo here mean?

Reply:
@bwplotka still curious what you meant here

Reply:
I meant this symbol table is technically wrong - no empty string there. But fine,  as no one checks. Perhaps would be nice to add empty string to teach others how it should look like in Prod (: